src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf

```diff
@@ -68,39 +68,39 @@
             </trans-unit>
             <trans-unit id="17">
                 <source>The mime type of the file is invalid ({{ type }}). Allowed mime types are {{ types }}.</source>
-                <target>Mime tip datoteke nije validan ({{ type }}). Dozvoljeni mime tipovi su {{ types }}.</target>
+                <target>MIME tip datoteke nije validan ({{ type }}). Dozvoljeni MIME tipovi su {{ types }}.</target>
```

Comment:
not sure if this should be caps

Reply:
Also not sure. I've found it in some books on Serbian it's written in full caps, which is why I changed it.